### ISSUE TITLE
Unable to play audio in Paper Edit view

### ISSUE THREAD

**Issue by leihelondon** [bbc/digital-paper-edit-client#117]
**Issue:**
Clicking on the play button in the Programme Script view on the right doesn't play audio. 

**To replicate:**
Sign in as admin and go to the follow links and click the play button in the Programme Script view on the right
https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/486aMNX8J6xHazceIicY/paperedits/8bm9IS4OdJSLpu7IJWVY
https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/MBjdsSOVTlB5v7VdJf2C/paperedits/y5NC4fyrDnoi6ugZLZi7

<img width="1680" alt="Screen Shot 2021-05-27 at 16 29 57" src="https://user-images.githubusercontent.com/4565059/119798550-27339180-bed3-11eb-96b9-c35114e93be6.png">

**Comment by allishultes** [comment]
FYI I have tested this on my end, on the test environment, with a .wav and a .mp3 that I uploaded yesterday. Both play back. @leihelondon I'll try to find this project in the screenshot but if you have examples that aren't working that would be helpful!

See: https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/486aMNX8J6xHazceIicY/paperedits/pDrXCm3YlYHT7lAuuLgx

**Comment by leihelondon** [comment]
> FYI I have tested this on my end, on the test environment, with a .wav and a .mp3 that I uploaded yesterday. Both play back. @leihelondon I'll try to find this project in the screenshot but if you have examples that aren't working that would be helpful!
> 
> See: https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/486aMNX8J6xHazceIicY/paperedits/pDrXCm3YlYHT7lAuuLgx

Just updated the ticket.

**Comment by allishultes** [comment]
Both programme scripts linked to above use expired and/or deleted media content. From looking into this, I believe that if **any** of the media files referenced in the paper edit do not exist, none of the media will play in the preview. 

Re: https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/MBjdsSOVTlB5v7VdJf2C/paperedits/y5NC4fyrDnoi6ugZLZi7

<img width="1440" alt="Screenshot 2021-05-27 at 11 10 03" src="https://user-images.githubusercontent.com/22618458/119808676-4387fc00-bedc-11eb-9451-10b5ad0e1e25.png">

Only the "Roland Hughes" file was not expired. When I created [a new programme script](https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/MBjdsSOVTlB5v7VdJf2C/paperedits/S5hrpaAD7lNmSJuzzZoh) in this project, and only made selections from this transcript, the media played back perfectly (see link).

Re: https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/486aMNX8J6xHazceIicY/paperedits/8bm9IS4OdJSLpu7IJWVY

The programme script referenced a deleted media file, and therefore did not play back (this would've been confusing to debug as everything I've uploaded in this project has the same name). 

To test whether it would play back the files that do exist, I tried adding a new paper-cut segment to this project, from a transcript that had its media stored still. It still did not play back, which is why I believe the playback function will not work if any of the files are expired.

Suggestion: we should clean up test so that we aren't diagnosing bugs that likely won't effect users doing an under-60-day trial. I can also add some notes about this to the user manual.
